PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -37,5 +46,6 @@
     print("WARNING: More than one graveyard set found.")
 
 #Remove the graveyard EntitySet from the data.
-groups_to_write = [group_set for group_set in group_categories if group_set not in graveyard_sets]
-mb.write_file("no_graveyard.h5m", output_sets=groups_to_write)
+groups_to_write = [group_set for group_set in group_categories 
+                   if group_set not in graveyard_sets]
+mb.write_file(args.outputfile, output_sets=groups_to_write)
```

Comment:
Seconded. Here, it would be good to check whether or not that optional argument needs to be supplied. 

I had to look up how to do this, because I couldn't remember :mag: :thinking: 

So here's the answer I found for convenience: https://stackoverflow.com/questions/30487767/check-if-argparse-optional-argument-is-set-or-not